src/main/java/org/gridsuite/study/server/SupervisionController.java

```diff
@@ -95,4 +95,13 @@ public ResponseEntity<Long> getIndexedTombstonedEquipmentsCount() {
     public ResponseEntity<Long> deleteStudyIndexedEquipmentsAndTombstoned(@PathVariable("studyUuid") UUID studyUuid) {
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(supervisionService.deleteStudyIndexedEquipmentsAndTombstoned(studyUuid));
     }
+
+    @DeleteMapping(value = "/studies/{studyUuid}/nodes/builds")
+    @Operation(summary = "Invalidate node builds for the given study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "all built nodes for the given study have been invalidated")})
+    public ResponseEntity<Void> invalidateAllNodeBuilds(@PathVariable("studyUuid") UUID studyUuid) {
+        supervisionService.invalidateAllNodeBuilds(studyUuid);
```

Comment:
Invoke directly the `StudyService`

Reply:
Will add logs in `SupervisionService::invalidateAllNodeBuilds` and pass `StudyService::invalidateBuild` public